.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Dialyzer
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      MIX_ENV: test
+    name: OTP ${{matrix.pair.otp}} / Elixir ${{matrix.pair.elixir}}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - pair:
+              otp: 25.0
+              elixir: 1.14.3
+    steps:
+      - uses: actions/checkout@v2
+      - uses: erlef/setup-elixir@v1
+        with:
+          otp-version: ${{matrix.pair.otp}}
+          elixir-version: ${{matrix.pair.elixir}}
+      - name: "Compile and run Dialyzer"
+        run: |
+          mix do deps.get, deps.compile, compile
+          mix ci
```

Comment:
This would require [caching PLT files](https://hashrocket.com/blog/posts/build-the-ultimate-elixir-ci-with-github-actions), otherwise, each push would be validated forever.